.vale/styles/RedHat/ObviousTerms.yml

```diff
@@ -0,0 +1,16 @@
+---
+extends: existence
+level: suggestion
+# link:
+message: "Consider not documenting %s because it is self-explanatory."
+scope: list
+# source:
+action:
+  name: remove
+tokens:
+  - User
```

Comment:
I see where you are going with this, but won't this lead to a lot of false positives?

Reply:
I was thinking about this. We could flag the terms only when bolded (scope: strong?). That should limit occurrences to UI procedures. Or we could flag "**Description** field".

Reply:
Just changed scope to "strong".